toonz/sources/toonz/xshnoteviewer.cpp

```diff
@@ -627,7 +627,15 @@ void NoteArea::updateButtons() {
 void NoteArea::flipOrientation() { m_viewer->flipOrientation(); }
 
 void NoteArea::onXsheetOrientationChanged(const Orientation *newOrientation) {
-  m_flipOrientationButton->setText(newOrientation->caption());
+  //  m_flipOrientationButton->setText(newOrientation->caption());
+
+  QIcon flipOrientationIcon = createQIcon("fliporientation");
```

Comment:
For the same reason, I think this line should be just `QIcon flipOrientationIcon;` as well.